src/services/color/color.service.ts

```diff
@@ -8,69 +8,61 @@ import {
 
 export class ColorService {
 
-    private _html = '<div class="primary-color"></div>' +
-    '<div class="accent-color"></div>' +
-    '<div class="secondary-color"></div>' +
-    '<div class="alternate1-color"></div>' +
-    '<div class="alternate2-color"></div>' +
-    '<div class="alternate3-color"></div>' +
-    '<div class="vibrant1-color"></div>' +
-    '<div class="vibrant2-color"></div>' +
-    '<div class="grey1-color"></div>' +
-    '<div class="grey2-color"></div>' +
-    '<div class="grey3-color"></div>' +
-    '<div class="grey4-color"></div>' +
-    '<div class="grey5-color"></div>' +
-    '<div class="grey6-color"></div>' +
-    '<div class="grey7-color"></div>' +
-    '<div class="grey8-color"></div>' +
-    '<div class="chart1-color"></div>' +
-    '<div class="chart2-color"></div>' +
-    '<div class="chart3-color"></div>' +
-    '<div class="chart4-color"></div>' +
-    '<div class="chart5-color"></div>' +
-    '<div class="chart6-color"></div>' +
-    '<div class="ok-color"></div>' +
-    '<div class="warning-color"></div>' +
-    '<div class="critical-color"></div>';
+    private _defaultTheme = [
```

Comment:
Same as the AngularJS version, let's pull these from a common file and make sure that the objects are exposed so that developers can assign whichever color set that they prefer.

(Also I was thinking that "theme" might not be a good term to use here, since we use theme to describe the full set of styling changes, rather than just colors.)